tests/core/Jalasoft.TeamUp.Projects.Core.Tests/InvitationsServiceTests.cs

```diff
@@ -108,5 +109,20 @@ public void GetInvitationsByProjectId_InvitationsDontExist_EmptyArray()
             // Assert
             Assert.Empty(result);
         }
+
+        [Fact]
+        public void PatchInvitationByProjectId_InvitationIsValid_InvitationUpdated()
+        {
+            // Arrange
+            Invitation stubInvitation = StubInvitation.GetStubInvitation();
+
+            this.mockRepository.Setup(repository => repository.GetById(Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"))).Returns(stubInvitation);
+
+            // Act
+            var result = this.service.UpdateInvitation(stubInvitation);
+
+            // Assert
+            Assert.IsType<Invitation>(result);
+        }
     }
 }
```

Comment:
please add unit tests for `InvitationValidator`

Reply:
done